frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -88,69 +98,72 @@ class PodStatus extends React.Component<PodStatusProps, PodStatusState> {
       title = '',
       subTitle = '',
     } = this.props;
-    const { vData, updateOnEnd } = this.state;
+    const { vData, updateOnEnd, tipIndex } = this.state;
 
     const tooltipEvent: any = showTooltip
       ? [
           {
-            target: 'data',
             eventHandlers: {
-              onMouseOver: () => {
-                return [
-                  {
-                    target: 'labels',
-                    mutation: () => {
-                      return { active: true };
-                    },
-                  },
-                ];
-              },
-              onMouseOut: () => {
-                return [
-                  {
-                    target: 'labels',
-                    mutation: () => {
-                      return { active: false };
-                    },
-                  },
-                ];
+              onMouseOver: (e, hoverSlice) => {
+                this.setState({ tipIndex: hoverSlice.index });
               },
             },
           },
         ]
       : undefined;
+
+    const tipContent = (
+      <div className="odc-pod-status-tooltip">
+        {vData[tipIndex] && (
+          <React.Fragment>
+            <span
+              key={1}
```

Comment:
Any reason for the `key`? Constant React keys typically don't have a ton of use-cases.

Reply:
Bah, leftover... thanks